frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteRepositoryDiscoverer.java

```diff
@@ -57,7 +58,8 @@ public void enqueue(Path macheteFilePath) {
       public void run(ProgressIndicator indicator) {
         LOG.debug("Running automatic discover task");
         val gitMacheteRepositoryInstantiateResult = Try
```

Comment:
nice. btw does it make sens to use a usual try catch instead of vavr here?

Reply:
You're right, I changed it now under a try-related PR:
https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1325/commits/484071a44d12df530a8a9fb3c8938607392dc0d2